resources/apt-packages.txt

```diff
@@ -0,0 +1,38 @@
+# utilities
+git
+mercurial
+pkg-config
+wget
+# debian-provided interpreters
+python-pip
+python2.7
+python2.7-dev
```

Comment:
Yes, I was thinking mainly the *-dev packages.  Docker is weird in that there doesn't seem to be a clear notion between build and install packages.  I know it seems really weird that people might rely on these, but experience tells me if you supply, some engineer somewhere will rely on it.  As a rule, I try to export as little as possible.